src/main/resources/akka.conf

```diff
@@ -0,0 +1,54 @@
+akka {
+
+  http {
+    server.parsing.max-content-length = 10g
+    host-connection-pool  {
+      max-connections   = 16
+      max-connections   = ${?AKKA_HTTP_MAX_CONNECTIONS}
+      max-open-requests = 64
+      max-open-requests = ${?AKKA_HTTP_MAX_OPEN_REQUESTS}
+    }
+  }
+
+  loggers = ["akka.event.slf4j.Slf4jLogger"]
+  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
+  log-dead-letters = off
+  loglevel = INFO
+  loglevel = ${?LOG_LEVEL}
+
+  extensions += "akka.cluster.ddata.DistributedData"
+
+  actor {
+    provider = "akka.cluster.ClusterActorRefProvider"
+
+    serializers {
+      circeEvent = "ch.epfl.bluebrain.nexus.kg.serializers.Serializer$EventSerializer"
+    }
+
+    serialization-bindings {
+      "ch.epfl.bluebrain.nexus.kg.resources.Event"   = circeEvent
```

Comment:
we also need to serialize additional data types: `Command`, `State`, `Msg` (from sourcing-akka), `Stop` from StreamCoordinator (service-indexing)

Reply:
this will be addressed in #342